internal/service/streamprocessor/resource_schema.go

```diff
@@ -53,8 +53,8 @@ func ResourceSchema(ctx context.Context) schema.Schema {
 			"state": schema.StringAttribute{
 				Optional:            true,
 				Computed:            true,
-				Description:         "The state of the stream processor.",
-				MarkdownDescription: "The state of the stream processor.",
+				Description:         "The state of the stream processor. Used to start or stop the Stream Processor. Valid values are `CREATED`, `STARTED` or `STOPPED`  When a Stream Processor is created without specifying the state, it will default to `CREATED` state.",
```

Comment:
Schema descriptions are automatically generated from the OpenAPI spec. This change is to improve the generated documentation of the `state` attribute in the resource documentation. The descriptions will be improved in the OpenAPI spec as part of [CLOUDP-267769](https://jira.mongodb.org/browse/CLOUDP-267769)